ROADMAP.md

```diff
@@ -0,0 +1,38 @@
+# KF Serving 2019 Roadmap
+## Q2 2019
+
+### Core CUJs
+* High Level Interfaces
```

Comment:
Can you please add a couple sentences outlining the motivation for each high level CUJ here.

Reply:
ack